pkg/api/customization/project/project_actions.go

```diff
@@ -289,6 +290,15 @@ func (h *Handler) setPodSecurityPolicyTemplate(actionName string, action *types.
 		return fmt.Errorf("could not convert: %v", providedPSP)
 	}
 
+	if ok {
+		if _, err := h.PSPTemplateLister.Get("", podSecurityPolicyTemplateName); err != nil {
+			if apierrors.IsNotFound(err) {
+				return httperror.NewAPIError(httperror.InvalidBodyContent, "podSecurityPolicyTemplate not found")
```

Comment:
nit: might be good to include psp template name after psptemplate, otherwise this error is probably pretty similar to the raw one you get from the lister

Reply:
something like `fmt.Sprintf("podSecurityPolicyTemplate [%s] not found", podSecurityTemplateName)`

Reply:
fixed